src/FullBuild/Core/BuildArtifacts.fs

```diff
@@ -19,12 +19,15 @@ open Graph
 open Collections
 
 
-let Publish (graph : Graph) (tagInfo : Baselines.TagInfo) =
-    let tag = tagInfo.Format()
+let Publish (graph : Graph) =
+    let baselines = Baselines.from graph
+    let baseline = baselines.GetPulledBaseline() |> Option.get
```

Comment:
GetPulledBaseline --> GetSourcesBaseline ?
otherwise there will be missing repositories in baseline